### ISSUE TITLE
Function (tables - variants)

### ISSUE THREAD

**Issue by Araurelius** [SpaceTown-Developers/Lemon-Gate#91]
Hello,
in this moment I'm writing pretty complex code destinated for "import" using (like dll) usings. So, I had a problem with return values that functions give, for example:
I have 
function newControl() {return MyTable;}

and I can't use it like:

((table)newControl())[< table value >, < type >] = /\* new value */.

Immedately you may suggest me to load this kind of table to variable:
table NewTable = (table)newControl();
But, as I specified before, this code is being used as imported code, so, local variables won't interact with variables that I have in another file. 
In conclusion, I think that return values your functions have, are read-only. 
Can you solve this problem or send me some idea how to deal with it?
